src/pages/publication/PublicationList.tsx

```diff
@@ -73,8 +73,8 @@ const PublicationList: FC<PublicationListProps> = ({ publications }) => {
         </TableHead>
         <TableBody>
           {publications
-            .filter(publication => publication.status !== PublicationStatus.PUBLISHED)
-            .map(publication => (
+            .filter((publication) => publication.status !== PublicationStatus.PUBLISHED)
+            .map((publication) => (
```

Comment:
burde kanskje hatt paginering og filtrere bort duplikater?
![Skjermbilde 2020-04-02 kl  14 08 49](https://user-images.githubusercontent.com/6595982/78247644-7fe3e600-74eb-11ea-993a-5b8be6ff81c9.png)

Reply:
eller backend bør kanskje filtrere bort duplikater. men paginering kan vi sikkert gjøre?

Reply:
det der ser egentlig ut til å være en backend-bug? at det ble opprettet 9stk identiske i stedet for 1?

Reply:
paginering må vi nok ha ja, men tenker det er greit å ta senere?

Reply:
joda, det er nok sant. kanskje litt tidlig å opprette bug på det i og med at de holder på med det nå. men ja - kan ta paginering senere 👍 lager du en task på det?